diff --git a/dist/framework/index.js b/dist/framework/index.js
index eac6d59e8eb094cd5082627921238762960e448f..2d3b6c1a515dbc9970858cdefbbd4018b6497b70 100644
--- a/dist/framework/index.js
+++ b/dist/framework/index.js
@@ -29,7 +29,7 @@ function FrameworkProvider({ Link, useRouter, useParams, usePathname, Image, chi
 	});
 }
 function usePathname() {
-	return use(FrameworkContext).usePathname();
+	return decodeURIComponent(use(FrameworkContext).usePathname());
 }
 function useRouter() {
 	return use(FrameworkContext).useRouter();
